echo_server.py

```diff
@@ -1,28 +1,149 @@
 #! /usr/bin/env python
 """Echo server in socket connection: receives and sends back a message."""
 import socket
+import os
+import mimetypes
+import binascii
+
+
+def response_ok(tuple):
+    """Return byte string 200 ok response."""
+    # u"HTTP/1.1 200 OK\r\nContent-Type: text/plain\r\nContent-length: 18\r\n\r\neverything is okay".encode('utf-8')
+    okay = u"HTTP/1.1 200 OK\r\nContent-Type: "
+    c_len = u"\r\nContent-length: "
+    crlf = u"\r\n\r\n"
+    return "{}{}{}{}{}{}".format(okay, tuple[0], c_len, len(tuple[1]), crlf, tuple[1])
```

Comment:
colorization should be warning you about the fact that you are using a builtin as an argument.  Not a great idea.  What's in this "tuple"  Maybe name it in a way that makes that clear?